### PR TITLE
fix(docs): update docs

### PR DESCRIPTION
**Pull request by UcheSylvester**

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -50,13 +54,50 @@ const result = pm.validate('MyPassword123'); // { valid: false, messages: ['must
...
+Each validator receives a `passwordRule` _(number)_ argument, which defines the number of expected characters associated with the validator.
+Consider the following example:
 
-NB: The only exception to this is the `NoSpaceCharacterValidator` which requires no arguments and simply ensures the password contains no spaces.
+```ts
+// Standard API
+import { MinLengthValidator } from '@password-validator/core';
+
+const minLength = new MinLengthValidator(10);
+const result = minLength.validate("eihi2kd#");
```
Comment (Theshedman) [inline README.MD:86]: Remember that the users don't need to use the validators directly except through the `PasswordValidatorManager` class. Please update the usage sample to something like below:

```ts
import { PasswordValidatorManager } from '@password-validator/core';

const pm = PasswordValidatorManager.standard();

const minLength = new MinLengthValidator(10);
pm.register(minLength);

const result = pm.validate("eihi2kd#");
```

File: README.MD [hunk 2]
```diff
@@ -50,13 +54,50 @@ const result = pm.validate('MyPassword123'); // { valid: false, messages: ['must
...
+
+// Fluent API
+import { PasswordValidatorManager } from '@password-validator/core';
+
+const result = PasswordValidatorManager.fluent()
+  .min(10)
+  .validate("eihi2kd#");
+```
+
+The `new MinLengthValidator(10)` or `PasswordValidatorManager.fluent().min(10)`  defined above means that the passowrd must be atleast __10 characters__ long to be considered valid.
```
Comment (Theshedman) [inline README.MD:98]: Update the reference here as well

**Review by Theshedman — CHANGES_REQUESTED** [review]

File: README.MD [hunk 3]
```diff
@@ -74,22 +115,22 @@ pm.register(minLength, maxLength);
 
 const result = pm.validate('MyPassword123!*') // Throws PasswordValidatorConflictException --> `minLength cannot be greater than maxLength`
```
Comment (Theshedman) [inline README.MD:116]: Conflict checks and errors happen at the point of registering the validators.

`pm.register(minLength, maxLength);`

and not at the point of validating the password. Please fix it.

**Review by Theshedman — CHANGES_REQUESTED** [review]

File: README.MD (hunk 1 above)
Comment (UcheSylvester) [reply README.MD:86]: I probably did this with sleep in my eyes. Apologies 😄

Comment (UcheSylvester) [reply README.MD:86]: scratch that! this actually uses pasword  manager: `const result = pm.validate("eihi2kd#")`, it's just using one validator. 

I wanted to use just one validator to explain `passwordRule` and its usage.

File: README.MD (hunk 3 above)
Comment (UcheSylvester) [reply README.MD:116]: Nice catch! Thanks

File: README.MD (hunk 2 above)
Comment (UcheSylvester) [reply README.MD:98]: This is still explain how to create validators

File: README.MD (hunk 1 above)
Comment (Theshedman) [reply README.MD:94]: I understand but you can still do that with the password validator manager

**Pull request merged**